Graph/lineMooring1DVar.m

```diff
@@ -159,9 +184,25 @@ function lineMooring1DVar(sample_data, varName, isQC, saveToFile, exportDir)
             set(hAxMooringVar, 'XLim', [xMin, xMax]);
             hold(hAxMooringVar, 'on');
             
-            % reverse the colorbar as we want surface in red and bottom in blue
-            cMap = colormap(hAxMooringVar, jet(lenSampleData));
-            cMap = flipud(cMap);
```

Comment:
Did you think this was not useful? Would still be relevant with parula. My idea was it would be easier to identify which instrument is meant to be close to surface or bottom depending on the color of their line. The deeper the more blue.